schemas/com.snowplowanalytics.mobile/list_item_view/jsonschema/1-0-0

```diff
@@ -0,0 +1,32 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an event tracked when an item is displayed in a list",
+	"self": {
+		"vendor": "com.snowplowanalytics.mobile",
+		"name": "list_item_view",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"index": {
+			"type": "integer",
+			"description": "Index of the list item on the screen",
+			"minimum": 0,
+			"maximum": 65535
+		},
+		"items_count": {
+			"type": [
+				"integer",
+				"null"
+			],
+			"description": "Total number of list items on the screen",
+			"minimum": 0,
+			"maximum": 65535
+		}
```

Comment:
Dumb question, more just for my knowledge - would these be 0 indexed so there might be 5 items but the index of the last item will be 4?

Reply:
No I think this is the right way to do it!